pkg/ingester/ingester.go

```diff
@@ -96,8 +96,9 @@ type Config struct {
 }
 
 // SetClientConfig sets clientConfig in config
-func (cfg *Config) SetClientConfig(clientConfig client.Config) {
+func (cfg *Config) SetClientConfig(clientConfig client.Config, limits validation.Limits) {
 	cfg.clientConfig = clientConfig
+	cfg.limits = limits
```

Comment:
This should either be put in a second function (`SetLimits`) or the function should be renamed

Reply:
I'm going to propose a new method named `SetSharedConfigs` that accepts the same two params. An alternative would be to accept a single `distributor.Config` instance and let the method pull out what it wants (but this seems to further couple the ingestor.Config and the distributor.Config and I prefer the two params).